### PR TITLE
Add shellcheck to pre-commit config

### PR DESCRIPTION
**Pull request by jpopelka**
And a few other polishing commits.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: "2"
 
 services:
   redis:
-    image: quay.io/centos7/redis-5-centos7
+    image: quay.io/sclorg/redis-6-c9s
```
Comment (TomasTomecek) [inline docker-compose.yml:5]: sneaky

File: files/run_tests.sh [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 set -x
 
+# shellcheck source=/dev/null
 source /src/files/setup_env_in_openshift.sh
```
Comment (TomasTomecek) [inline files/run_tests.sh:6]: huh, is this telling shellcheck to not try to access that file?

**Review by TomasTomecek — APPROVED** [review]
Your completely unrelated commits are always so amusing to review 😁

File: files/run_tests.sh (hunk 2 above)
Comment (jpopelka) [reply files/run_tests.sh:6]: Yes, there's no such file in the repository (it is in the image/container) so shellcheck can't access it.
Should I add a comment with a link to https://www.shellcheck.net/wiki/SC1091 ?

**Comment by jpopelka** [comment]
> Your completely unrelated commits are always so amusing to review

I see, just tell me if you'd rather prefer 5 separate PRs :-)

**Review by lbarcziova — APPROVED** [review]
:+1:

File: files/run_tests.sh (hunk 2 above)
Comment (TomasTomecek) [reply files/run_tests.sh:6]: it would definitely be welcoming to people who see this line for the first time, thank you for the explanation

**Review by FrNecas — APPROVED** [review]

**Pull request merged**